naurffxiv/src/app/ultimates/dsr.js

```diff
@@ -0,0 +1,226 @@
+                    <li>Tank Rot Priority: WAR - GNB - - - PLD - DRK</li>
+                    <li>If AM is used for Wroth, chains go to the wall and red Ignores go closer to the middle (under Hraesvelgr)</li>
+                    <li>WB2 in the toolvox is outdated as static WB2 is now used (see image)</li>
+                    <li>Melee tethers in WB2 will not fully stretch, they will still live</li>
+                </ul>
+                <h3>Phase 7: Dragon-King Thordan</h3>
+                <ul>
+                    <li><a href="https://ff14.toolboxgaming.space/?id=448767858029561&preview=1" target="blank">Toolbox</a></li>
+                    <li>MRH trios, G1 (numbers) then G2 (letters)</li>
+                    <li>3-3-2 stacks: G1 NW, G2 NE, tanks south</li>
+                    <li>6-1-1 stacks: G1 and G2 NW, Kitchen Sink Tank NE, Invuln Tank S</li>
+                    <li>Ignore slides 33/34 (Slide 34 shows party positioning for 6-1-1 AMs however)</li>
+                </ul>
+            </div>
+
+            <div className="resources">
+                <h2>General Resources</h2>
+                <h3>Mitigation Sheets</h3>
+                <ul>
+                    <li><a href="https://docs.google.com/spreadsheets/d/1YJX933Fe6MeNv06QVQfRUI3oJVfCvm4OqOyLCSjcAJk/edit?usp=sharing" target="blank">NAUR Mitigation Sheet</a></li>
+                    <li><a href="https://docs.google.com/document/d/18i2ElKFtkO4c1MLIUP-hY-YXNLa4iTi8jIlrLF2dMb8/edit?usp=sharing" target="blank">Double Shield Mitigation Sheet</a></li>
+                    <li><a href="https://docs.google.com/spreadsheets/d/1o3H2j46Q5ZVKB6OxhOO8__DV2zRues1etuOabxlJA_8/edit?usp=sharing" target="blank">DPS Mitigation Sheet</a></li>
+                </ul>
+                <h3>Cheat Sheets & Images</h3>
+                <ul>
+                    <li><a href="https://raw.githubusercontent.com/naurffxiv/assets/main/Toolbox%20Resources/DSR/StaticWB2Positions.png" target="blank">Static WB2 Positions</a></li>
+                    <li><a href="https://raw.githubusercontent.com/naurffxiv/assets/main/Toolbox%20Resources/DSR/StaticWB2Positions.png" target="blank">DotH Movement</a></li>
+                    <li><a href="https://raw.githubusercontent.com/naurffxiv/assets/main/Toolbox%20Resources/DSR/DSRExaflareMovement.png" target="blank">Triple Exaflare Dodges</a></li>
+                </ul>
+                <h3>DSR Simulations</h3>
+                <ul>
+                    <li><a href="https://www.xivsim.com/dru/" target="blank">XIV Sim</a></li>
+                    <li><a href="https://caffinate.itch.io/exaflares" target="blank">Exaflare Helper</a></li>
+                    <li><a href="https://github.com/WCGH/Dragonsong-Sim/releases" target="blank">Solo Sim for P3, Wrath, DotH, Dragons, Exaflares</a></li>
+                </ul>
+                <h3>Helpful Macros</h3>
+                <ul>
+                    <li>P1 Setup Macro:      
+      
+     </li>

```

Comment:
we should fix this macro

Reply:
is the macro not correct? my understanding is that it will only display correctly in game.

Reply:
ok i wasnt sure if that was the case, but we can just leave it for now